### ISSUE TITLE
Compatibilities of node versions

### ISSUE THREAD

**Issue by biko8** [nothingisdead/npm-kraken-api#43]
Please mention that this package is **only** compatible with node version 8. Because it uses async/wait functions.

**Comment by OpinionatedGeek** [comment]
I see this too. I like this API's approach, and it would be great if it could be compatible with Node 6.10 - that's the highest available Node version for my AWS stuff.

**Comment by johnelkins** [comment]
The node mean ami is severely outdated. I ended up spinning up an ubuntu 16.04 and installing node 8 myself. Might be worth the effort.

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by johnelkins** [comment]
BTW... the api had been working perfectly for me. Thank you.

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by johnelkins** [comment]
Async/ await is a much better approach to promises. Makes for a very clean and succinct code base. I refactored an entire AWS api to use async/await and it was def worth it.

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by OpinionatedGeek** [comment]
Yeah, I'd be happy to update my node version if I could but I'm using AWS Lambda and it only goes as high as Node 6.10.

I'm not against using async/await, BTW. I know other projects use Babel to compile the JavaScript they write to a lower version of JavaScript that more environments can use. How difficult would it be to put that in this project? (I did try using it to compile all my dependencies in node_modules, so that I could use this API, but it wasn't a great experience and I'm told it's not a great idea.)

Many thanks,

Geoff
